repoload/repoload.py

```diff
@@ -151,13 +168,84 @@ def get_changes_for_topic(topic):
     return changes
 
 
-def get_change(number):
-    changes, _ = query(URL, "%s" % (number,))
+def get_change(number, options=None):
+    changes, _ = query(URL, "%s" % (number,), options=options)
     if len(changes) != 1:
         raise Exception("Cannot find change for id %s" % (number,))
     return changes[0]
 
 
+def show_change(args):
+    # Show information about a change id
+    number = args.value
+    change = get_change(number, options=[GerritOptions.CommitMessage,
+                                         GerritOptions.CurrentPatchSet,
+                                         GerritOptions.PatchSets,
+                                         GerritOptions.Comments])
+    print_change(change)
+
+
+def print_change(change):
+    w = TextWrapper(width=72, initial_indent="\t", subsequent_indent="\t")
+    c = change
+    print("Project:\t{} @ {} ".format(c["project"], c["branch"]))
+    print("Status:\t\t{}".format(c["status"]))
+    if c["open"] == "true":
+        print("Cr is open")
+
+    print("Change id:\t{}".format(c["id"]))
+    print("Change number:\t{}".format(c["number"]))
+
+    print("Created at\t{}".format(datetime.fromtimestamp(c["createdOn"])))
+    print("Last updated\t{}".format(datetime.fromtimestamp(c["lastUpdated"])))
+    print("Url\t\t{}".format(c["url"]))
+
+    print("Change subject:\t{}".format(c["subject"]))
+    print("Change owner:\t{} ({})".format(c["owner"]["name"], c["owner"]["email"]))
+
+    print('\nCommit Message:\n')
+    for line in str.splitlines(c["commitMessage"]):
+        print(w.fill(line))
+
+    print('\n')
+    p = c["currentPatchSet"]
+    print("PatchSet\t{}:".format(p['number']))
+    print("Kind\t\t{}".format(p['kind']))
+    print("Ref\t\t{}".format(p['ref']))
+    print("Revision\t{}".format(p["revision"]))
+
+    parents = " \n".join(p["parents"])
+    print("Parents\t\t{}".format(parents))
+    print("Created\t\t{}".format(datetime.fromtimestamp(p["createdOn"])))
+
+    print("Size\t\t+{} / -{}".format(p["sizeInsertions"], p["sizeDeletions"]))
+
+    if 'approvals' in p:
+        print("Approved by:")
+        for approval in p['approvals']:
+            print("    {} : {} [{}]".format(approval['value'],
+                                            approval['by']['name'],
+                                            approval['description']))
+
+    # TODO: Also show the comments on files
+    print('\n')
+    print('Comments:')
```

Comment:
hmm, I think there should be maybe two separate commands or the command should accept a `--comment` argument. One show the basic information of a CR and one to show all information.